### PR TITLE
Add support for superscript and subscript text

### PR DESCRIPTION
**Pull request by kristianvalind**
RTF supports superscript text using `\super` and subscript text using `\sub`. `\nosupersub` is used to revert any of these modifiers. This pull request adds support for parsing and rendering these tags, along with a test case.

### CODE REVIEW

File: src/rtfjs/renderer/RenderChp.ts [hunk 1]
```diff
@@ -75,5 +75,15 @@ export class RenderChp {
...
         el.style.fontSize = Math.floor(this._chp.fontsize / 2) + "pt";
+
+        if (this._chp.supersubscript === 1) {
+            el.style.verticalAlign = "super"
+        } else if (this._chp.supersubscript === -1) {
+            el.style.verticalAlign = "sub"
+        }
+
+        if (this._chp.supersubscript === 1 || this._chp.supersubscript === -1) {
+            el.style.fontSize = Math.floor((this._chp.fontsize / 2) - 2) + "pt"
```
Comment (zoehneto) [inline src/rtfjs/renderer/RenderChp.ts:86]: According to the spec we should `shrinks point size according to font information`, I haven't found how that is supposed to work though. As the result here matches the Libreoffice output quite well, I'd leave it as is for now.

**Review by zoehneto — CHANGES_REQUESTED** [review]
Please also include the updated build in your PR (`npm run build`).

**Comment by zoehneto** [comment]
I just updated the dependencies on the master branch, so please rebase before updating the build.

File: test/rtf-test-files/superscript-subscript/expected.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div style="min-height: 12pt; text-align: left;">
```
[the remark this answers is not in the document]
Comment (kristianvalind) [reply test/rtf-test-files/superscript-subscript/expected.html:1]: I'm not sure either, I created the test file in TextEdit.app on Mac OS, and changed no formatting expect for superscript and subscript. I regenerated the test files, but still get the same formatting.

**Comment by kristianvalind** [comment]
> I just updated the dependencies on the master branch, so please rebase before updating the build.

I accidentally merged the changes instead. Please let me know if I should work around this somehow or if it's acceptable the way it is.

File: test/rtf-test-files/superscript-subscript/expected.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (zoehneto) [reply test/rtf-test-files/superscript-subscript/expected.html:1]: Perhaps it automatically formats any file you view? It's correct now.

**Comment by zoehneto** [comment]
> > I just updated the dependencies on the master branch, so please rebase before updating the build.
> 
> I accidentally merged the changes instead. Please let me know if I should work around this somehow or if it's acceptable the way it is.

That works as well.

**Review by zoehneto — APPROVED** [review]

**Comment by zoehneto** [comment]
Thanks for making the fixes, I'll merge this PR now.

**Pull request merged**